docs/sources/enterprise/request-security.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Request security"
+description = "Grafana Enterprise request security"
+keywords = ["grafana", "security", "enterprise"]
+weight = 110
++++
+
+# Request security
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+Request security introduces ways to limit requests from the Grafana server and focuses on requests generated by users.
+
+Example:
+- datasource metric queries
+- alert notifications
+
+This can be used to limit access to internal systems that the server Grafana runs on can access but that users of Grafana should not be able to access. This feature does not affect traffic from the Grafana users browser.
+
+Note: While request security works with backend plugins, it is possible to create a backend plugin that bypasses this security.
+
+## IP and hostname blocking
+
+It's possible to limit requests based on hostname and/or ip addresses in two ways.
```

Comment:
```suggestion
It is possible to limit requests based on a hostname, an IP address, or both.
```
I removed "two ways" here to make this more future proof wrt doc bugs, and when in doubt, readers can indeed count.

Reply:
Notice how I worked around that "and/or" construction.

Reply:
Nice, thank you.